JeMPI_Apps/JeMPI_UI/src/services/ApiClient.ts

```diff
@@ -29,12 +27,46 @@ import {
   PatientRecord
 } from 'types/PatientRecord'
 import { Notifications } from 'types/Notification'
+import { Config } from 'config'
+import axios from 'axios'
+import { getCookie } from '../utils/misc'
 
-const client = config.shouldMockBackend ? moxios : axiosInstance
+export class ApiClient {
+  client!: AxiosInstance
+
+  async init(config: Config) {
+    if (process.env.NODE_ENV === 'test') {
+      this.client = moxios
+    } else {
+      const axiosInstance = axios.create({
+        withCredentials: true,
+        baseURL: config.apiUrl
```

Comment:
Isnt this always set. i.e if we are having to go the backup, we have a serious problem, and we should allow it to fail.